src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+.form-search {
+  margin-top: 20px;
+}
+
+.form-search__input {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+  width: 100%;
+  background: rgb(255, 255, 255);
+  border: 1px solid rgb(225, 231, 237);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: 300;
+}
+
+.form-search__input:hover {
+  border-color: rgb(102, 102, 102);
+}
+
+.form-search__input:focus {
+  border-color: rgb(51, 51, 51);
+  outline: none;
+}
+
+.form-search__input::placeholder {
+  color: rgb(61, 78, 97);
+}
+
+.form-search__small .form-search__input {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.form-search__big .form-search__input {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.form-search__input_icon {
+  display: block;
+  position: absolute;
+}
```

Comment:
I think you should add `position: relative` on parent container to position it easier

Reply:
added relative to parent in solution2